db/schema.rb

```diff
@@ -11,11 +11,32 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170123205547) do
+ActiveRecord::Schema.define(version: 20170223174042) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "episode_images", force: :cascade do |t|
```

Comment:
Ok, almost. Now we have podcast_images and episode_images.
However, there are 2 kinds of podcast image: the itunes_image, and the feed_image, which are subclasses of podcast_image